src/store/fileUploadStore.spec.js

```diff
@@ -0,0 +1,78 @@
+import { createLocalVue } from '@vue/test-utils'
+import storeConfig from './storeConfig'
+import Vuex from 'vuex'
+import { cloneDeep } from 'lodash'
+
+import globalStore from '../store/index'
+
+// workaround for files using the store globally
+jest.mock('../store/index', () => jest.fn())
```

Comment:
we can't put an object in here without redefining it completely, so the closest we can mock is a function, but the users of the module expect an object, not a function